### PR TITLE
Code cleanup of Dragging module

### PR DESCRIPTION
**Pull request by jokoho48**
Code cleanup of Dragging module.

### CODE REVIEW

File: addons/dragging/functions/fnc_canCarry.sqf [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
+ *
+ * Public: No
  */
 #include "script_component.hpp"
 
-private ["_unit", "_target"];
-
-_unit = _this select 0;
-_target = _this select 1;
+private
```
Comment (MikeMatrix) [inline addons/dragging/functions/fnc_canCarry.sqf]: `private` should be removed here.

File: addons/dragging/functions/fnc_carryObjectPFH.sqf [hunk 2]
```diff
@@ -1,4 +1,16 @@
-// by commy2
+/*
+ * Author: commy2
+ *
+ * PFH for Crarry Object
```
Comment (MikeMatrix) [inline addons/dragging/functions/fnc_carryObjectPFH.sqf]: `Crarry` should be `Carry`

File: addons/dragging/functions/fnc_carryObjectPFH.sqf [hunk 3]
```diff
@@ -6,16 +18,15 @@
 #endif
 
 private ["_unit", "_target"];
```
Comment (MikeMatrix) [inline addons/dragging/functions/fnc_carryObjectPFH.sqf]: `private` no longer needed.

File: addons/dragging/functions/fnc_handleAnimChanged.sqf [hunk 4]
```diff
@@ -1,4 +1,20 @@
-// by commy2
+/*
+ * Author: commy2
+ *
+ * Handle the Animaion for a Unit for Dragging Module
```
Comment (MikeMatrix) [inline addons/dragging/functions/fnc_handleAnimChanged.sqf]: `Animaion` should be `animation`

File: addons/dragging/functions/fnc_handleAnimChanged.sqf [hunk 5]
```diff
@@ -1,4 +1,20 @@
...
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Animaion <STRING>
```
Comment (MikeMatrix) [inline addons/dragging/functions/fnc_handleAnimChanged.sqf]: Same here

File: addons/dragging/functions/fnc_handleKilled.sqf [hunk 6]
```diff
@@ -1,9 +1,22 @@
-// by commy2
+/*
+ * Author: commy2
+ *
+ * Handle the Kill of the Dragger
```
Comment (MikeMatrix) [inline addons/dragging/functions/fnc_handleKilled.sqf]: Use `death of the dragger` here instead. Makes more sense.

File: addons/dragging/functions/fnc_handlePlayerChanged.sqf [hunk 7]
```diff
@@ -1,10 +1,23 @@
-// by commy2
+/*
+ * Author: commy2
+ *
+ * Handle the Event if player is Changed
```
Comment (MikeMatrix) [inline addons/dragging/functions/fnc_handlePlayerChanged.sqf]: Maybe use `Handle player changes.` instead.

File: addons/dragging/functions/fnc_handlePlayerWeaponChanged.sqf [hunk 8]
```diff
@@ -1,10 +1,23 @@
-// by commy2
+/*
+ * Author: commy2
+ *
+ * Handle the Weapon Changed Event
```
Comment (MikeMatrix) [inline addons/dragging/functions/fnc_handlePlayerWeaponChanged.sqf]: Maybe use `Handle weapon changes.` here.

File: addons/dragging/functions/fnc_handleUnconscious.sqf [hunk 9]
```diff
@@ -1,17 +1,29 @@
-// by commy2
+/*
+ * Author: commy2
+ *
+ * Handle the Unconscious of a Unit while Dragging
```
Comment (MikeMatrix) [inline addons/dragging/functions/fnc_handleUnconscious.sqf]: Maybe use `Handle loss of consciousness while dragging.` here.

File: addons/dragging/functions/fnc_handleUnconscious.sqf [hunk 10]
```diff
@@ -1,17 +1,29 @@
...
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_unit] call ace_dragging_fnc_handlePlayerWeaponChanged;
```
Comment (MikeMatrix) [inline addons/dragging/functions/fnc_handleUnconscious.sqf]: wrong function name

**Comment by MikeMatrix** [comment]
Nice work

**Comment by jonpas** [comment]
There shouldn't be any space in the function documentation between author and description.

File: addons/dragging/functions/fnc_handleScrollWheel.sqf [hunk 11]
```diff
@@ -3,38 +3,38 @@
...
- * Return value:
- * Handled or not. (Bool)
+ * Return Value:
+ * Handled or not. <BOOL>
+ *
+ * Public: No
  */
 #include "script_component.hpp"
 
+private ["_unit", "_carriedItem", "_position", "_maxHeight", ]
```
Comment (Glowbal) [inline addons/dragging/functions/fnc_handleScrollWheel.sqf]: Comma on the end but not new private element.

File: addons/dragging/functions/fnc_getWeight.sqf [hunk 12]
```diff
@@ -1,54 +1,45 @@
...
-    } foreach (_items select 0);
-    
-    _weight
-};
-_totalWeight = ([getMagazineCargo _object, "CfgMagazines"] call _fnc);
-_totalWeight = _totalWeight + ([getItemCargo _object, "CfgWeapons", "ItemInfo"] call _fnc_Extra);
-_totalWeight = _totalWeight + ([getWeaponCargo _object, "CfgWeapons", "WeaponSlotsInfo"] call _fnc_Extra);
-_totalWeight = _totalWeight + ([getBackpackCargo _object, "CfgVehicles"] call _fnc);
+        // Multiply mass with amount of items and add the mass to the total weight.
+        _totalWeight = _totalWeight + (getNumber ((call _getConfigCode) >> "mass") * (_count select _forEachIndex));
```
Comment (Glowbal) [inline addons/dragging/functions/fnc_getWeight.sqf]: It is not necessary to wrap the config path into code and call it. You can ditch the `{}` on line 35 through 38 and the call on this line.

**Pull request merged**